fixtures/common.py

```diff
@@ -4,15 +4,37 @@
 from selenium.webdriver.chrome.options import Options
 from selenium import webdriver
 
-
 @pytest.fixture(scope='session')
 def driver():
-    chrome_options = Options()
-    if settings.HEADLESS:
-        chrome_options.add_argument('--headless')
-        chrome_options.add_argument('--disable-gpu')
-    chrome_options.add_argument('--window-size=512,512')
-    driver = webdriver.Chrome(chrome_options=chrome_options)
+    if settings.SAUCELABS_USERNAME:
+        if settings.BROWSER == 'ie11':
+            desired_cap = {
+                'platform': "Windows 10",
+                'browserName': "internet explorer",
+                'version': "11.103",
+                }
```

Comment:
The closing braces should be indented one level left.

Reply:
Ok, but we need to document that.